src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEEventSymbol.cs

```diff
@@ -92,11 +92,16 @@ private enum Flags : byte
                 _eventType = metadataDecoder.GetTypeOfToken(eventType);
             }
 
+            TypeSymbol originalEventType = _eventType;
+            _eventType = DynamicTypeDecoder.TransformType(_eventType, 0, handle, moduleSymbol);
+            // Dynamify object type if necessary
+            _eventType = _eventType.AsDynamicIfNoPia(_containingType);
```

Comment:
Aleksey, I noticed that DynamicTypeDecoder.TransformType assumes the presence of customModifiers (haveCustomModifierFlags: true on line 72). This doesn't seem harmful, as the decoder logic handles the case of zero custom modifiers properly, but it's a bit awkward. 
We should discuss.